.changeset/nasty-emus-vanish.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperlane-xyz/registry': minor
```

Comment:
Since something was removed (fantom addresses), this should be major.

Reply:
they're being replaced with our own addresses in the upcoming PR which addresses for these new chains, would that still mean this is a minor?

Reply:
If we don't release in between then it's probably. Tbh it's probably fine either way, just not _strictly_ ideal